topic0KeyPoint1/src/oop2_2/Book.java

```diff
@@ -0,0 +1,73 @@
+package oop2_2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Book {
+
+	private String name;
+	private List<Author> listOfAuthors = new ArrayList<Author>();
+	private double price;
+	private int qty;
+
+	public Book(String name, List<Author> listOfAuthors, double price) {
+
+		this.name = name;
+		this.listOfAuthors = listOfAuthors;
+		this.price = price;
+
+	}
+
+	public Book(String name, List<Author> listOfAuthors, double price, int qty) {
+
+		this.name = name;
+		this.listOfAuthors = listOfAuthors;
+		this.price = price;
+		this.qty = qty;
+	}
+
+	public double getPrice() {
+		return price;
+	}
+
+	public void setPrice(double price) {
+		this.price = price;
+	}
+
+	public int getQty() {
+		return qty;
+	}
+
+	public void setQty(int qty) {
+		this.qty = qty;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public List<Author> getListOfAuthors() {
+		return listOfAuthors;
+	}
+
+	public String toString() {
+		StringBuilder stringBuilder = new StringBuilder();
+
+		stringBuilder.append("Book[name=" + this.getName() + ",authors={");
```

Comment:
You are using a StringBuilder to make the String, but you are also concatenating the Strings using the + operator. Choose one